cmd/stacker/check.go

```diff
@@ -26,12 +27,23 @@ func doCheck(ctx *cli.Context) error {
 		return errors.Wrapf(err, "couldn't get kernel info")
 	}
 
-	log.Infof(kernel)
+	log.Infof("os/kernel: %s", kernel)
 
 	if err := os.MkdirAll(config.RootFSDir, 0700); err != nil {
 		return errors.Wrapf(err, "couldn't create rootfs dir for testing")
 	}
 
+	// internally there are many checks to avoid symlinks
+	evalp, err := filepath.EvalSymlinks(config.RootFSDir)
+	if err != nil {
+		return errors.Wrapf(err, "%s: unable to evaluate path for symlinks", config.RootFSDir)
+	}
+
+	if evalp != config.RootFSDir {
+		return errors.Errorf("%s: roots dir (--roots-dir) path uses symbolic links, use %s instead", config.RootFSDir, evalp)
```

Comment:
Is it generally deemed safer to use %q for escaping paths?

Reply:
Fixed.